File: lacticio/src/main/java/com/laticinios/laticinio/model/Produto.java
package com.laticinios.laticinio.model;

import jakarta.persistence.*;
import lombok.*;
@AllArgsConstructor
@NoArgsConstructor
@Data
@Entity
@Table(name = "tb_produto")
public class Produto {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;
    private Double temperatura;
    private Double peso;
    private String tipo;
    private Integer quantidade;
}

File: src/model/Produto.ts
export class Produto {
    id!: number;
    endereco!: string;
    tipo!: string;
    quantidade!: number;
    
    

    constructor(id: number, endereco: string, tipo: string, quantidade: number) {
        this.id = id;
        this.endereco = endereco;
        this.tipo = tipo;
        this.quantidade = quantidade;
    }
}